src/parse.js

```diff
@@ -276,6 +290,87 @@ export class CellParser extends Parser {
   }
 }
 
+export class TemplateCellParser extends CellParser {
+  constructor({tag, raw, ...options}, ...args) {
+    super(options, ...args);
+    this.tag = tag;
+    this.raw = raw;
+  }
+  parse() {
+    return this.parseCell(this.startNode());
+  }
+  parseCell(node) {
+    // Based on acorn’s q_tmpl. We will use this to initialize the 
+    // parser context so our `readTemplateToken` override is called.
+    // `readTemplateToken` is based on acorn's `readTmplToken` which
+    // is used inside template literals. Our version allows backQuotes.
+    const o_tmpl = new TokContext(
+      "`", // token
+      true, // isExpr
+      true, // preserveSpace
+      parser => readTemplateToken.call(parser) // override
+    );
```

Comment:
This should be static, at the top level rather than declared within the parseCell. We don’t need to create a new one each time parseCell is called.